ext/js/display/search-display-controller.js

```diff
@@ -66,11 +66,11 @@ export class SearchDisplayController {
         /** @type {boolean} */
         this._clipboardMonitorEnabled = false;
         /** @type {import('clipboard-monitor').ClipboardReaderLike} */
-        const clipboardReader = {
+        this.clipboardReaderLike = {
```

Comment:
What does `Like` mean here?

Reply:
Makes it clear that this isnt an instance of the clipboardReader class and doesn't have all the function of it but has similar base function. Similar to an "arraylike" type where theres an array that might be able to do the array storage stuff but it might not be able to do `push()` or `pop()` for example.

Reply:
In other languages you would accomplish this by typing an interface but ig js doesn't have this pattern